<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AngelScript IDE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
    background-color: rgb(255, 255, 255);
}

QMenuBar::item {
    background: transparent;
}

QMenuBar::item:selected {
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}

QStatusBar {
    background-color: rgb(40, 40, 40);
}

QDockWidget::title {
    text-align: center;
    background: rgb(0, 180, 255);
}

/*QSplitter::handle {
	background-color: rgb(100, 100, 100);
}

QSplitter::handle:horizontal {
    width: 2px;
}

QSplitter::handle:vertical {
    height: 2px;
}*/

QToolBar {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(215, 215, 215, 255), stop:1 rgba(255, 255, 255, 255));
	padding: 0px;
	border-bottom: transparent;
}

QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #C2C7CB;
}
QTabWidget::tab-bar {
left: 5px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1E1E1, stop: 0.4 #DDDDDD, stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
border: 2px solid #C4C4C3;
border-bottom-color: #C2C7CB; /* same as the pane color */
border-top-left-radius: 4px;
border-top-right-radius: 4px;
min-width: 8ex;
padding: 2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #fafafa, stop: 0.4 #f4f4f4, stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #C2C7CB; /* same as pane color */
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionPublish"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="actionRename"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionStepOver"/>
    <addaction name="actionStepInto"/>
    <addaction name="actionStepOut"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuWorkspace">
    <property name="title">
     <string>Workspace</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTabbed"/>
    <addaction name="separator"/>
    <addaction name="actionCustomize"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuWorkspace"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/new_folder.png</normaloff>:/menubar/icons/new_folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Project...</string>
   </property>
   <property name="toolTip">
    <string>Create new project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/open_folder.png</normaloff>:/menubar/icons/open_folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
   <property name="toolTip">
    <string>Open project file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/save_all.png</normaloff>:/menubar/icons/save_all.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save All</string>
   </property>
   <property name="toolTip">
    <string>Save all open files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit AngelScriptIDE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPublish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Publish...</string>
   </property>
   <property name="toolTip">
    <string>Publish project</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/run.png</normaloff>:/toolbar/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run program</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
   <property name="toolTip">
    <string>Project options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
   <property name="toolTip">
    <string>Show editor files in cascaded windows</string>
   </property>
  </action>
  <action name="actionTabbed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabbed</string>
   </property>
   <property name="toolTip">
    <string>Show editor files in tabs</string>
   </property>
  </action>
  <action name="actionCustomize">
   <property name="text">
    <string>Customize...</string>
   </property>
   <property name="toolTip">
    <string>Customize workspace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About ASIDE</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Context...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Change editor settings</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="toolTip">
    <string>Close the current project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/pause.png</normaloff>:/toolbar/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause game</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/stop.png</normaloff>:/toolbar/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop game</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/toolbar/icons/restart.png</normaloff>:/toolbar/icons/restart.png</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="toolTip">
    <string>Restart game</string>
   </property>
  </action>
  <action name="actionBrowse_Project">
   <property name="text">
    <string>Open in Explorer</string>
   </property>
   <property name="toolTip">
    <string>Open current folder in Explorer</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/open_file.png</normaloff>:/menubar/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="toolTip">
    <string>Open script file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/save_file.png</normaloff>:/menubar/icons/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menubar/icons/new_file.png</normaloff>:/menubar/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New File...</string>
   </property>
   <property name="toolTip">
    <string>Create script file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/debugger/icons/dbgstepover.png</normaloff>:/debugger/icons/dbgstepover.png</iconset>
   </property>
   <property name="text">
    <string>Step Over</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/debugger/icons/dbgstepinto.png</normaloff>:/debugger/icons/dbgstepinto.png</iconset>
   </property>
   <property name="text">
    <string>Step Into</string>
   </property>
  </action>
  <action name="actionStepOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/debugger/icons/dbgstepout.png</normaloff>:/debugger/icons/dbgstepout.png</iconset>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRun</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startApplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>interruptExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestart</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>restartExec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseProject</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createProject()</slot>
  <slot>openProject()</slot>
  <slot>startApplication()</slot>
  <slot>closeProject()</slot>
  <slot>interruptExec()</slot>
  <slot>restartExec()</slot>
  <slot>stopExec()</slot>
  <slot>saveFile()</slot>
  <slot>saveAllFiles()</slot>
  <slot>openFile()</slot>
  <slot>newFile()</slot>
  <slot>showSettings()</slot>
 </slots>
</ui>
